xrpl/utils/txn_parser/utils/get_balance_changes.py

```diff
@@ -0,0 +1,147 @@
+"""Helper functions for `get_balance_changes`."""
```

Comment:
If these helper functions are only being used by `get_balance_changes`, and you won't need them for the other methods, you could put them in the same `get_balance_changes.py` file above as the main function